src/uu/wc/src/wc.rs

```diff
@@ -210,33 +223,33 @@ impl TotalWhen {
     }
 }
 
-#[derive(Debug)]
+#[derive(Debug, Error)]
```

Comment:
`thiserror` is not doing a whole lot here except here in my opinion, but it's a big deal, because we already use `thiserror` elsewhere.

Reply:
> When thiserror fits, I think it's a pretty nice method of creating errors.

Yeah, I kinda agree. We should probably get some project-wide standard on this. At the moment it's mostly that most utils use no library and then there's some `thiserror` and even `quick-error` in some utils, which I think I a bit annoying. I think it's easiest to converge on not using a library, because we rarely need the fancy features of `thiserror`. Here it's also just implementing `Display`.

With errors you usually end up cloning because they tend to escape the scope where they were created.

Btw, I made an edit to my original comment, because I made a very significant typo:
> but it's a big deal,

should have been

> but it's not a big deal,

Sorry if that caused some confusion :)

Reply:
Heh, I think I only noticed that the third time I read it :)